server/secret.go

```diff
@@ -1,32 +1,46 @@
 package server
 
 import (
+	"net/http"
+
 	"github.com/drone/drone/model"
 	"github.com/drone/drone/router/middleware/session"
 	"github.com/drone/drone/store"
 
 	"github.com/gin-gonic/gin"
 )
 
+func GetSecrets(c *gin.Context) {
+	repo := session.Repo(c)
+	secrets, err := store.GetSecretList(c, repo)
+
```

Comment:
Do we need the ID? How should I build that? Iterate through the secrets and convert them?